azurerm/resource_arm_function_app_test.go

```diff
@@ -1024,27 +1112,85 @@ resource "azurerm_storage_account" "test" {
 resource "azurerm_app_service_plan" "test" {
   name                = "acctestASP-%[1]d"
   location            = "${azurerm_resource_group.test.location}"
+  kind                = "Linux"
   resource_group_name = "${azurerm_resource_group.test.name}"
 
   sku {
     tier = "Standard"
     size = "S1"
+  }
+	
+	properties {
```

Comment:
Could we use spaces instead of tabs for indentation here?

Reply:
Should've done what I just did from the beginning... 😄
`cmd + shift + p` -> `Convert Indentation to Spaces`